src/components/Tailwind/Panels/GeoBlocked.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PageWarning from '../Layout/PageWarning'
+import { GeofenceCountry, useGeofence } from '../../../halo-hooks/useGeofence'
+
+interface GeoBlockedProps {
+  children: React.ReactNode
+  country: GeofenceCountry
+}
+
+const GeoBlocked = ({ children, country }: GeoBlockedProps) => {
+  const { loading, rejected } = useGeofence(country)
```

Comment:
So for this one, `useGeofence` param can be an array instead so we can block multiple countries too